jenkins/rpm/v2/post_install_script.sh

```diff
@@ -0,0 +1,33 @@
+RES_AGENT="/usr/lib/ocf/resource.d/seagate"
+HA_INSTALL_DIR=/opt/seagate/cortx/ha
+BIN_DIR=${HA_INSTALL_DIR}/bin
+mkdir -p ${BIN_DIR} ${RES_AGENT}
+
+HA_SETUP=/usr/lib/python3.6/site-packages/ha/setup/ha_setup.py
```

Comment:
/usr/lib/python3.6/site-packages/ha/setup/ha_setup.py

is it fix
i think there are multiple site-packages path but not sure how to make sure it always go to same path

Reply:
Fixed for the python version this project is using but there must be better way to it, will check.

Reply:
done